<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_currentUnit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 17pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="iconCurrent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_addMachine">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_durationUnit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 17pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="iconDuration">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="inputDuration">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(1, 1, 1);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_machineName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>15</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 17pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_machineName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(9, 9, 9);
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="inputCurrent">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(0,0,0)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_help">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_caution">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>button_addMachine</zorder>
    <zorder>label_currentUnit</zorder>
    <zorder>iconCurrent</zorder>
    <zorder>label_durationUnit</zorder>
    <zorder>iconDuration</zorder>
    <zorder>inputDuration</zorder>
    <zorder>label_machineName</zorder>
    <zorder>text_machineName</zorder>
    <zorder>inputCurrent</zorder>
    <zorder>button_help</zorder>
    <zorder>label_caution</zorder>
   </widget>
   <widget class="QListWidget" name="listWidget_machine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(2, 2, 2);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Execute">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_trash">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_openTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_closeTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 16pt &quot;Georgia&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="time_openTime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="time_closeTime">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_backCover">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
